hls-visualization.ipynb

```diff
@@ -0,0 +1,720 @@
+    "\n",
+    "[Landsat 8 Bands and Combinations Blog](https://www.l3harrisgeospatial.com/Learn/Blogs/Blog-Details/ArtMID/10198/ArticleID/15691/The-Many-Band-Combinations-of-Landsat-8)\n",
+    "\n",
+    "[HLSS30 Dataset Landing Page](https://lpdaac.usgs.gov/products/hlss30v002/)\n",
+    "\n",
+    "[Sentinel 2 Bands and Combinations Blog](https://gisgeography.com/sentinel-2-bands-combinations/)\n",
+    "\n",
+    "[CQL2 STAC-API Examples](https://github.com/radiantearth/stac-api-spec/tree/master/fragments/filter#examples)\n",
+    "\n",
+    "- Author: Alexandra Kirk\n",
+    "- Last Updated Date: June 30, 2022"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import requests\n",
+    "from folium import Map, TileLayer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Exploring the stac catalog - curated parameters for investigating specific hurricane events with the dynamic tiler and custom band combinations\n",
+    "Notebook examples focus on HLS S30 data for Hurricane Ida but Hurricane Maria and LS30 parameters are provided below for further exploration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Endpoints\n",
+    "stac_base_url = \"https://staging-stac.delta-backend.com\"\n",
+    "raster_base_url = \"https://staging-raster.delta-backend.com\" \n",
+    "\n",
+    "# Harmonized Sentinel collection id and configuration info\n",
+    "s30_collection_id = \"hls-s30-002-ej-reprocessed\"\n",
+    "s30_swir_assets = [\"B12\",\"B8A\",\"B04\"]\n",
+    "s30_vegetation_index_assets = [\"B08\", \"B04\"]\n",
+    "s30_vegetation_index_expression = \"(B08-B04)/(B08+B04)\"\n",
+    "s30_vegetation_index_rescaling = \"0,1\"\n",
+    "s30_vegetation_index_colormap = \"rdylgn\"\n",
+    "\n",
+    "# Harmonized Landsat collection id and map configuration info\n",
+    "l30_collection_id = \"hls-l30-002-ej-reprocessed\"\n",
+    "l30_swir_assets = [\"B07\",\"B05\",\"B04\"]\n",
+    "l30_ndwi_expression = \"(B03-B05)/(B03+B05)\"\n",
+    "l30_ndwi_assets = [\"B03\", \"B05\"]\n",
+    "l30_ndwi_rescaling = \"0,1\"\n",
+    "l30_ndwi_colormap = \"spectral\"\n",
+    "\n",
+    "# Search criteria for events in both HLS Events collections\n",
+    "maria_bbox = [-66.167596, 17.961538, -65.110098, 18.96772]\n",
+    "maria_temporal_range = [\"2017-06-06T00:00:00Z\", \"2017-11-30T00:00:00Z\"]\n",
+    "\n",
+    "ida_bbox = [-90.932637, 29.705366, -89.766437, 30.71627]\n",
+    "ida_temporal_range = [\"2021-07-01T00:00:00Z\", \"2021-10-28T00:00:00Z\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### First search the STAC api for items for specific dates available\n",
+    "This search body can be used with both the STAC and raster APIs, we will restrict the temporal range for mosaicing"
```

Comment:
```suggestion
    "This search body can be used with both the STAC and raster APIs. We will restrict the temporal range for mosaicing."
```

Can we add a brief explanation for why we restrict the temporal range - is it just to limit the set of data included in the mosaic to something manageable?